Common/src/main/java/com/tabnineCommon/chat/commandHandlers/context/GetBasicContextHandler.kt

```diff
@@ -54,7 +56,113 @@ class GetBasicContextHandler(gson: Gson) : ChatMessageHandler<Unit, BasicContext
         if (metadata?.has("error") == true) {
             metadata = null
         }
-
-        return BasicContext(metadata)
+        return BasicContext(null, getPredominantWorkspaceLanguage(), metadata)
     }
 }
+
+fun getPredominantWorkspaceLanguage(
+    includeFilePredicate: (String) -> Boolean = { true }
+): String? {
+    val project = ProjectManager.getInstance().openProjects.firstOrNull() ?: return null
+    val fileIndex = FileBasedIndex.getInstance()
```

Comment:
pls, check that it is running in the correct context (read/write) https://plugins.jetbrains.com/docs/intellij/general-threading-rules.html#read-access

Reply:
I can make it run in read 
```kotlin
ApplicationManager.getApplication().runReadAction {
```